api/net/Areas/Admin/Models/DataSource/DataLocationModel.cs

```diff
@@ -8,6 +10,15 @@ namespace TNO.API.Areas.Admin.Models.DataSource;
 public class DataLocationModel : BaseTypeWithAuditColumnsModel<int>
 {
     #region Properties
+    /// <summary>
+    /// get/set -
+    /// </summary>
+    public DataLocationType LocationType { get; set; }
```

Comment:
Type provides the app the ability to make logic decisions (not implemented yet).